### ISSUE TITLE
Remove torrent immediately, but allow Undo

### ISSUE THREAD

**Issue by DiegoRBaquero** [webtorrent/webtorrent-desktop#970]
I'd like to make a feature request.

If you are removing many torrents from the client you have to click on the remove button in the confirmation dialog each time. It would be great if I could check a checkbox that said "Do not show this again" and just click on the X that removes the torrents.

What do you guys think?

**Comment by mathiasvr** [comment]
I actually included such a checkbox in my initial PR for the confirmation modals, but it seemed more clean not to have it. Another way to solve this is by having a preference as suggested in #853.

**Comment by feross** [comment]
The cleanest way to solve this, IMO, is to just remove it immediately, but to show a little "Undo" hover card at the bottom of the view for 5 seconds after deleting a torrent.

The thinking is: Do what the user wants immediately. Treat the user like an adult who knows what they're doing, so don't ask for confirmation constantly. Yet, provide a means for them to recover from mistakes.

Here's an example of how Inbox (by Google) implements this, in the bottom-left corner of the app.

<img width="505" alt="screen shot 2016-09-26 at 11 19 26 pm" src="https://cloud.githubusercontent.com/assets/121766/18862253/bb96dc84-843f-11e6-8432-7aaafc055d10.png">

**Comment by DiegoRBaquero** [comment]
@feross Damn, I love that idea. Would definitely leave preferences to be important stuff and never bother the user.

**Comment by feross** [comment]
@DiegoRBaquero Yeah, some things need preferences, but I think the right instinct is to push back and consider other solutions first. Once you add a preference it's really hard to remove. Let's try really hard to do the right thing, and use preferences as a last resort.

**Comment by mathiasvr** [comment]
@feross I like this idea, but I wonder if it would be enough to just use undo/cmd+z then?

**Comment by feross** [comment]
Hm, I think that a keyboard shortcut alone works as a first attempt. But I would want to play around with the little notification message at the bottom and see if that's helpful to the user. Maybe it's too much for a desktop app, but we should try it IMO.

Btw, for the "delete file data" option in the right click menu, we probably still want the confirmation dialog since that's not possible to reverse, and it's a less commonly used than just clicking the 'X'.
